outline.md

```diff
@@ -0,0 +1,59 @@
+## Growing an Inner Source Program
+
+45 min
+
+## Short Description
+
+"Inner source is a great idea! Go do inner source!"
+You're the dedicated champion for inner source in your company.
+What do you do now?
+How do you realistically affect the behavior of dozens or hundreds of teams to the point where robust and pervasive inner sourcing is a normal part of the way that engineering is done?
+
+This presentation shares principles, practical anecdotes, and relatable examples on this situation gleaned from experience at Nike.
+While not claiming all of the answers, it is insightful summary of over a year of work in the area.
+
+## Audience
+This presentation will be given at [the ISC Spring Summit](http://paypal.github.io/InnerSourceCommons/events/isc-spring-2018-agenda/).
+It is intended for those that are evangelizing inner source in a specific company.
+
+## Detailed description
+
+#### Outline
+
+An inner source program is the set of tools, documentation, training, and so forth provided so that any team can participate in inner source effectively as either a maintainer or contributor.
```

Comment:
`and so forth`  is what precisely? omit or be more specific?

Reply:
Early on in this I think you should articulate that any program should be self sustaing.  the impulse is to go whole hog and rollout big programs etc. I think the first point should be to clearly define or establish a vision or picture of what the new culture will look like then organically grow support until you hit your early majority at which phase two is rolling out the program guns.  A program should be able to support its own weight.  While not 100% complete some ideas can be found in my value stream proposal around [core body strength](https://www.notion.so/Core-Body-Strength-c23f26451b024b52a705dc6ba46b15a4) that might help support this.

There will be problems unique to your own organization.  These tend to come in two forms.  1) In companies where there is a flat organic mindset, people tend to employ the workaround strategy.  2) In companies where there is rigid top down authority based structure there tends to be more escalation based.  This means your vision should at least identify and work to attack these two problems.   In a lot of ways these are mirrored strategies, one coming from too much centralized governance and one coming from too little centralized governance.

You can explain that we have a more flat structure and are working to combat the workaround scenario in most cases, but we too have some places where interrupt and escalation based ivory towers need some attention too.   

I think this insight is super valuable and will provide a great backdrop for the talk.

Reply:
> and so forth is what precisely? omit or be more specific?

omitted.

> my value stream proposal around core body strength

<img width="678" alt="screen shot 2018-05-03 at 5 17 42 am" src="https://user-images.githubusercontent.com/9609562/39576008-63a10932-4e91-11e8-8580-db67bd102e39.png">

> There will be problems unique to your own organization. These tend to come in two forms

Very insightful comment!  I hadn't thought if it that way.  I like it!  I wish the Learning Path presentation came first so that I could use that terminology.  Do you think that the approach we'd take to rolling at the program would look that different based on trying to combat _workaround_ vs. _escalate_?

Reply:
I have access to the _core body strength_ page in _notion_ now.

Reply:
It might be worth taking 5 minutes to highlight the terms for consistenecy

Reply:
I'm having trouble with this as it seems like a point that is orthogonal to the main content.  Still thinking of leaving it out (or at least wait until the presentation to see how it fits in).  If it's something you feel strongly about then let's meet to discuss?